sorc/checkout.sh

```diff
@@ -21,7 +21,8 @@ if [[ ! -d gsi.fd ]] ; then
     rm -f ${topdir}/checkout-gsi.log
     git clone --recursive gerrit:ProdGSI gsi.fd >> ${topdir}/checkout-gsi.log 2>&1
     cd gsi.fd
-    git checkout gfsda.v15.3.2
+    #git checkout gfsda.v15.3.2
```

Comment:
Iliana, to keep things clean please remove the commented out checkout line (line 24) and remove the text comment on the next line. We only need to update the version # for the tag. Commit those adjustments to your fork branch and they'll appear in here. Reply to this comment when done and I'll submit the PR , thanks!

Reply:
Cleaned up the comments and pushed. Thanks!

Reply:
Perfect thanks!